### PR TITLE
R pidkostelnii login passport configuration

### PR DESCRIPTION
**Pull request by roman454647**

### CODE REVIEW

File: config/passport.js [hunk 1]
```diff
@@ -1,14 +1,54 @@
-const JWTStrategy = require('passport-jwt').Strategy;
-const ExtractJWT = require('passport-jwt').ExtractJwt;
-const UserQuery = require('../db/query/UserQuery');
-require('dotenv').config();
+const JWTStrategy = require('passport-jwt').Strategy
+const ExtractJWT = require('passport-jwt').ExtractJwt
+const UserQuery = require('../db/query/UserQuery')
+const passport = require('passport')
+var LocalStrategy = require('passport-local').Strategy
```
Comment (InnaKutsik) [inline config/passport.js]: var ?

File: controllers/auth.controller.passport.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const jwt = require('jsonwebtoken')
+const passport = require('passport')
+
+require('dotenv').config()
+
+exports.authPassport = (req, res) => {
+  passport.authenticate('local', { session: false }, (err, user, info) => {
+    console.log(err)
```
Comment (InnaKutsik) [inline controllers/auth.controller.passport.js]: ?

File: config/passport.js (hunk 1 above)
Comment (roman454647) [reply config/passport.js]: now it const

File: controllers/auth.controller.passport.js (hunk 2 above)
Comment (roman454647) [reply controllers/auth.controller.passport.js]: removed

**Pull request merged**